packages/core/src/common/command.spec.ts

```diff
@@ -37,69 +37,6 @@ describe('Commands', () => {
         expect('abc').equals(result);
     });
 
-    it('should execute a given command, and add it to recently used', async () => {
```

Comment:
I think we should attempt to fix the test cases rather than remove them completely. 
We can replace `.registerCommand` calls by `.addRecentCommand` to test the functionality of actually adding a new command to the list of recently used commands.

Reply:
I think this can go into quick-command tests, do you agree?

Reply:
Will do.

Reply:
Tests pass. @vince-fugnitto you can review it now.

Reply:
> Tests pass. @vince-fugnitto you can review it now.

Don't forget to squash :)

Reply:
@vince-fugnitto done!